src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/NotificationsControllerTest.java

```diff
@@ -91,19 +116,34 @@ public void sendAssignToJudgeConfirmationEmail() throws Exception {
     public void shouldNotSendAssignToJudgeConfirmationEmail() throws Exception {
         buildCcdRequest(CCD_REQUEST_JSON);
         mockMvc.perform(post(ASSIGN_TO_JUDGE_URL)
-                .content(requestContent.toString())
-                .contentType(MediaType.APPLICATION_JSON_VALUE))
+                .header(AUTHORIZATION_HEADER, AUTH_TOKEN)
+                .content(requestContent)
+                .contentType(APPLICATION_JSON_VALUE))
                 .andExpect(status().isOk());
 
         verifyNoMoreInteractions(notificationService);
     }
 
+    @Test
+    public void sendAssignToJudgeConfirmationBulkPrintLetter() throws Exception {
+        buildCcdRequest(CCD_REQUEST_WITH_BULK_PRINT_LETTER_CONSENT_JSON);
+        mockMvc.perform(post(ASSIGN_TO_JUDGE_URL)
+            .header(AUTHORIZATION_HEADER, AUTH_TOKEN)
+            .content(requestContent)
+            .contentType(APPLICATION_JSON_VALUE))
+            .andExpect(status().isOk());
+
+        verify(assignedToJudgeBulkPrintService, times(1)).sendLetter(eq(AUTH_TOKEN), any(CaseDetails.class));
+        verifyNoMoreInteractions(notificationService);
+    }
+
     @Test
     public void sendConsentOrderMadeConfirmationEmail() throws Exception {
         buildCcdRequest(CCD_REQUEST_WITH_SOL_EMAIL_CONSENT_JSON);
         mockMvc.perform(post(CONSENT_ORDER_MADE_URL)
-                .content(requestContent.toString())
-                .contentType(MediaType.APPLICATION_JSON_VALUE))
+                .header(AUTHORIZATION_HEADER, AUTH_TOKEN)
```

Comment:
Whats the reasoning to why are we adding this?

Reply:
If you look at the changes in NotificationController.test you'll see we've added a header for the endpoints to require an authentication header: "@RequestHeader(value = AUTHORIZATION_HEADER) String authToken," so we have to add the auth header here to the all the relevant tests or else they'd fail